src/p5.dance.js

```diff
@@ -307,7 +307,12 @@ module.exports = class DanceParty {
     if (move === "next") {
       move = 1 + ((sprite.current_move + 1) % (ANIMATIONS[sprite.style].length - 1));
     } else if (move === "prev") {
-      move = 1 + ((sprite.current_move - 1) % (ANIMATIONS[sprite.style].length - 1));
+      //Javascript doesn't handle negative modulos as expected, so manually resetting the loop
+      if (sprite.current_move === 0) {
+        move = (ANIMATIONS[sprite.style].length - 1);
+      } else {
+        move = ((sprite.current_move - 1) % (ANIMATIONS[sprite.style].length));
```

Comment:
I don't think the modulus does anything for you at this point; I'd do one of the following to simplify:
```js
// No modulus at all
move = sprite.current_move - 1
if (move < 0) {
  move = ANIMATIONS[sprite.style].length - 1
}
// or always add the denominator to the numerator so it's never negative.
const animationCount = ANIMATIONS[sprite.style].length;
move = (animationCount + sprite.current_move - 1) % animationCount;
```